Library/Homebrew/test/gpg_spec.rb

```diff
@@ -13,7 +13,12 @@
         shutup do
           subject.create_test_key(dir)
         end
-        expect(dir/".gnupg/secring.gpg").to exist
+
+        if subject.version == Version.create("2.1")
```

Comment:
Maybe switch this around, so it falls back to the newer version in the `else` branch?